modyn/selector/internal/selector_strategies/new_data_strategy.py

```diff
@@ -50,8 +50,7 @@ def _init_storage_backend(self) -> AbstractStorageBackend:
             )
         else:
             raise NotImplementedError(
-                f'Unknown storage backend "{
```

Comment:
I am not even sure why this was not caught by CI - already running pytest modyn immediately fails for me locally because of this

Reply:
because the autoformat broke the line weirdly here. note that the {} scope is outside of the format string, this is not valid syntax